mlir/include/mlir/Dialect/Vector/IR/VectorOps.h

```diff
@@ -145,6 +145,10 @@ inline bool isReductionIterator(Attribute attr) {
 /// constant operations.
 SmallVector<int64_t> getAsIntegers(ArrayRef<Value> values);
 
+/// Returns the integer numbers in `foldResult`. `foldResult` is expected to
+/// be constant operations.
+int64_t getAsInteger(OpFoldResult foldResult);
```

Comment:
Is this function needed? I can't see any usages in patch. Also, MLIR laredy has `getConstantIntValue` in `StaticValueUtils.h` let's not duplicate :)

Reply:
Oops, probably in an earlier version. I added it because we also have `getAsIntegers`. We may have to do some cleanup here. Thanks!